schedulezoomtalks.py

```diff
@@ -33,9 +33,24 @@
 - Time slot: {{ meeting_start }}—{{ meeting_end }} UTC
 - Your personal Zoom meeting [login link]({{ meeting_zoom_link }})
 - Host key: {{ meeting_host_key }}
+- Please find the template you may use for the talk announcement at your institution below
```

Comment:
Could we move this to a separate sentence right above the "Your meeting information:" line? I think it shouldn't be part of the meeting info